apply/current_cluster.go

```diff
@@ -45,6 +54,8 @@ func getCurrentNodes() (*v1.Cluster, error) {
 		}
 		cluster.Spec.Nodes.IPList = append(cluster.Spec.Nodes.IPList, addr)
 	}
```

Comment:
Feel curious about "Count", why we need to keep this value? just len(...) is fine?

Reply:
User will edit this value in CLOUD mod